packages/ember-metal/tests/run_loop/onerror_test.js

```diff
@@ -20,11 +20,11 @@ QUnit.test('With Ember.onerror set, errors in Ember.run are caught', function ()
   var thrown = new Error('Boom!');
   var caught;
 
-  Ember.onerror = function(error) { caught = error; };
+  setOnerror(error => { caught = error; });
 
-  run(function() { throw thrown; });
+  run(() => { throw thrown; });
 
   deepEqual(caught, thrown);
 
-  Ember.onerror = undefined;
+  setOnerror(undefined);
```

Comment:
I realize that this was already like this, but we should probably reset in a tear down in case the test dies.